cf_remote/paths.py

```diff
@@ -7,7 +7,8 @@ def path_append(dir, subdir):
 
 
 def cfengine_dir(subdir=None):
-    return path_append("~/.cfengine/", subdir)
+    cf_remote_dir = ".cfengine" if "CF_REMOTE_DIR" not in os.environ else os.environ["CF_REMOTE_DIR"]
+    return path_append("~/{}/".format(cf_remote_dir), subdir)
```

Comment:
`~` means the user's home folder, we don't want that, we want them to be able to specify a path anywhere on the file system.

```suggestion
    override_dir = os.getenv("CF_REMOTE_DIR")
    if override_dir:
        return path_append(override_dir, subdir)
    return path_append("~/.cfengine/", subdir)
```